### PR TITLE
New simulator icons from Design

### PR DESCRIPTION
**Pull request by rickpasetto**
Updated icons from the design team for the simulators

### CODE REVIEW

File: MockKitUI/Assets.xcassets/iCGM.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "iCGM.pdf",
```
Comment (novalegra) [inline MockKitUI/Assets.xcassets/iCGM.imageset/Contents.json]: [nit] Perhaps `mockCGM`?

File: MockKitUI/Assets.xcassets/IAGC.imageset/Contents.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "images" : [
     {
+      "filename" : "IAGC.pdf",
```
Comment (novalegra) [inline MockKitUI/Assets.xcassets/IAGC.imageset/Contents.json]: [nit] Perhaps `mockPump`?

**Review by novalegra — APPROVED** [review]
LGTM, just a naming nit (I'm happy to keep the naming that's here if that's your preference)

File: MockKitUI/Assets.xcassets/iCGM.imageset/Contents.json (hunk 1 above)
Comment (rickpasetto) [reply MockKitUI/Assets.xcassets/iCGM.imageset/Contents.json]: these were the names of the files as given to me by Paul.

File: MockKitUI/Assets.xcassets/IAGC.imageset/Contents.json (hunk 2 above)
Comment (rickpasetto) [reply MockKitUI/Assets.xcassets/IAGC.imageset/Contents.json]: these were the names of the files as given to me by Paul.

**Review by mpangburn — APPROVED** [review]

File: MockKitUI/Assets.xcassets/iCGM.imageset/Contents.json (hunk 1 above)
Comment (ps2) [reply MockKitUI/Assets.xcassets/iCGM.imageset/Contents.json]: I agree mockCGM is better; these are not "iCGM".

File: MockKitUI/Assets.xcassets/IAGC.imageset/Contents.json (hunk 2 above)
Comment (ps2) [reply MockKitUI/Assets.xcassets/IAGC.imageset/Contents.json]: Same, this mock pump is not an implementation of IAGC or representative of IAGC in any way. It also doesn't look like a pump? Could we get design to take another crack at this? It should have a port, or something that at least hints at being pump-like.

**Review by ps2 — CHANGES_REQUESTED** [review]
The asset names should not involve the FDA names for devices meeting a standard that these plugins do not meet and are not attempting to meet, and I don't think the pump icon visually communicates pump at all.

File: MockKitUI/Assets.xcassets/iCGM.imageset/Contents.json (hunk 1 above)
Comment (rickpasetto) [reply MockKitUI/Assets.xcassets/iCGM.imageset/Contents.json]: The`imageset` is named `mockCGM`.  The name of the file is just the one from design.

**Pull request closed without merging**